test/e2e_node/image_credential_provider.go

```diff
@@ -30,7 +30,7 @@ import (
 	admissionapi "k8s.io/pod-security-admission/api"
 )
 
-var _ = SIGDescribe("ImageCredentialProvider [Feature:KubeletCredentialProviders]", func() {
```

Comment:
The fg is GAed in v1.26. Should we remove the feature tag in e2e node test? But there are some test ci for this and we should not change it now in my mind.

- I opened  https://github.com/kubernetes/test-infra/pull/29126 to cleanup the feature gate usage only.

Reply:
Is this expected to work in all clusters? Feature also implies tests that are not portable. This test is known to be problematic but I'm not sure re: portability